lib/config/presets/github/index.spec.ts

```diff
@@ -179,7 +180,13 @@ describe('config/presets/github/index', () => {
           content: Buffer.from('{"from":"api"}').toString('base64'),
         });
       expect(
-        await github.getPresetFromEndpoint('some/repo', 'default', undefined)
+        await github.getPresetFromEndpoint(
+          'some/repo',
+          'default',
+          undefined,
+          githubApiHost,
+          undefined
+        )
```

Comment:
```suggestion
        await github.getPresetFromEndpoint('some/repo', 'default', undefined)
```

This should help to fix coverage issue